debian/docs

```diff
@@ -0,0 +1 @@
+debian/NOTICE
```

Comment:
If you decide to introduce this file, you need to specify more items. Not having this file, makes `dh_installdocs` fall back to the default behaviour.

> By default, it will include all existing files in the top-level source directory that are called BUGS, README*, TODO etc.

You should at least add README.md files, and make sure we do not stop adding any documentation when we are trying to add the notice file

Reply:
But we don't have a README file for now. And it is out of the scope of the task. If we need a README file in the future, we can open a task, proritize it and after that I can happily add this file and add into the docs file.